crypto/hpke/hpke.c

```diff
@@ -841,6 +841,7 @@ OSSL_HPKE_CTX *OSSL_HPKE_CTX_new(int mode, OSSL_HPKE_SUITE suite, int role,
 
  err:
     EVP_CIPHER_free(ctx->aead_ciph);
+    OPENSSL_free(ctx->propq);
```

Comment:
I hope it takes care of NULL values as input.

Reply:
It does as far as I know

Reply:
It does